backend/src/services/mail_verification/send.service.ts

```diff
@@ -0,0 +1,42 @@
+import { ENV } from '@common/enums/app/app';
+import nodemailer from 'nodemailer';
+
+import { generateTokens } from './token.service';
+
+const transporter = nodemailer.createTransport({
+  host: ENV.MAIL.SMTP_HOST,
+  port: ENV.MAIL.SMTP_PORT,
+  secure: false,
+  auth: {
+    user: ENV.MAIL.SMTP_USER,
+    pass: ENV.MAIL.SMTP_PASSWORD,
+  },
+});
+
+async function sendActivationMail(to: string, link: string): Promise<void> {
+  await transporter.sendMail({
+    from: ENV.MAIL.SMTP_USER,
+    to,
+    subject: 'Account activation on the website ' + ENV.MAIL.API_URL,
+    text: '',
+    html: `
```

Comment:
Looks dirty 
First, you can provide this HTML to variable, but it is also not good. 
Try to look if it is possible to create HTML template page, for example user_varification.html or any format.
Second, don`t use ${link} twice. links in mails looks ugly. Just make simple text or button would be better